code/WorkInProgress/ItemSpecials.dm

```diff
@@ -1085,8 +1085,19 @@
 				var/mob/living/carbon/human/H = hit
 				H.do_disorient(src.stamina_damage * mult, weakened = 10)
 
-			hit.TakeDamage("chest", 0, rand(2 * mult,5 * mult), 0, DAMAGE_BLUNT)
-			hit.bodytemperature += 4 * mult
+			if (ismob(hit))
+				var/mob/M = hit
+				M.TakeDamage("chest", 0	, rand(2 * mult,5 * mult), 0, DAMAGE_BLUNT)
+				M.bodytemperature += 4 * mult
+			else if (iscritter(hit))
+				var/obj/critter/C = hit
+				C.health -= 10
+				C.check_health()
+			else //bot
+				var/obj/machinery/bot/B = hit
+				B.health -= 10
+				if (B.health <= 0)
+					B.explode()
```

Comment:
this is not a safe way to handle damage. 
* This bypasses other procs to wake up hibernating critters up and is exploitable
* This bypasses critter vulnerabilities and defenses
* You should not be handling deaths in the weapon special